src/NServiceBus.Core/DelayedDelivery/DoNotDeliverBefore.cs

```diff
@@ -8,22 +8,22 @@
     public class DoNotDeliverBefore : DelayedDeliveryConstraint
     {
         /// <summary>
-        /// The actual time when the message can be available to the recipient.
-        /// </summary>
-        public DateTime At { get; }
-
-        /// <summary>
-        /// Initializes a new insatnce of <see cref="DoNotDeliverBefore"/>.
+        /// Initializes a new insatnce of <see cref="DoNotDeliverBefore" />.
         /// </summary>
         /// <param name="at">The earliest time this message should be made available to its consumers.</param>
         public DoNotDeliverBefore(DateTime at)
         {
-            if (at <= DateTime.UtcNow)
+            if (at.ToUniversalTime() <= DateTime.UtcNow)
```

Comment:
@bording can you elaborate why we should use TimeZoneInfo instead? By glancing at the docs, I see that TZI works better with historic DateTime, which in this case isn't allowed anyway?